doc/devel/user-cheatsheets/matplotlib-for-tableau-users.rst

```diff
@@ -8,12 +8,96 @@ Matplotlib for Tableau users
 Introduction
 --------------
 
+Tableau is Business Intelligence tool that provides a highly customizable dashboard to preview your data, being highly customizable. 
+Although the usecase is slightly different, the visualization principles stay the same, with the same idioms and best practice, but a
+different environment and API.
 
 Some key differences
 ---------------------
 
 Basics
 --------
 
+When using matplotlib you need to import the package into your program. For this tutorial we import the pyplot package from matplotlib. It is also recommended to use NumPy for your calculations.
+
+::
+
+    import numpy as np
+    from matplotlib import pyplot as plt
+
+Now we can use the different functions provided by numpy and pyplot by calling np and plt respectively.
+
+For the following plots we will use the following data stored in an array as upposed to the cells in an Excel sheet.
+
+::
+
+    x = [1,2,3,4,5,6,7,8,9,10] # In matplotlib you need to also define the x-axis
+    y = [10,1,9,2,8,3,7,4,6,5] # Mockup data for visualising the different plots
+    z = [1,10,2,9,3,8,4,7,5,6] # An extra dataset for particular plots
+
+    theta = np.linspace(0,2*np.pi,10) # Array with length 10 and a range from 0 to 2*pi
+
+
+The most basic plot would look like the following in written code:
+
+::
+
```

Comment:
Did we decide cheatsheets should use the pyplot interface?  I'd prefer the explicit interface (`ax.plot`).  

I don't know where we decided cheatsheets should go, but definitely not `doc/devel`!  

I wonder if these blocks would be better as `:: plot` directives so the user can see the result..  

I'm not sure how much tutorial we should have in here (its pretty repetitive with the rest of our docs), versus just having the table...

Reply:
Hi @jklymak, I found @chrissshe working in /doc/users/cheatsheets/ here: https://github.com/matplotlib/matplotlib/pull/16565. Would that be a better location?

Reply:
I rewrote most of it to reference other guides that might be helpful when coming from Excel together with some relevance to Excel to get people started. I also changed the location from devel to users.